pkg/collect/helm.go

```
@@ -158,3 +167,10 @@ func helmReleaseInfoByNamespaces(releaseInfo []ReleaseInfo) map[string][]Release
 
 	return releaseInfoByNamespace
 }
+
+func getHelmValues(actionConfig *action.Configuration, releaseName string, revision int) map[string]interface{} {
+	getAction := action.NewGetValues(actionConfig)
+	getAction.Version = revision
+	helmValues, _ := getAction.Run(releaseName)
```

Please return this error so it can be stored in the support bundle